doc/classes/Input.xml

```diff
@@ -38,6 +38,35 @@
 				Adds a new mapping entry (in SDL2 format) to the mapping database. Optionally update already connected devices.
 			</description>
 		</method>
+		<method name="clear_joy_motion_calibration">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<description>
+				Clears the calibration information for the specified joypad's motion sensors, if it has any and if they were calibrated.
+				See [method start_joy_motion_calibration] for an example on how to use joypad motion sensors and calibration in your games.
+				[b]Note:[/b] This feature is only supported on Windows, Linux, and macOS.
+			</description>
+		</method>
+		<method name="enable_joy_accelerometer">
```

Comment:
I'm curious as to why the method was renamed from `set_joy_accelerometer_enabled()` (as seen in the demo project) to `enable_joy_accelerometer()`. The naming seems less Godot-y to me, especially since you can pass `false` to disable it.

The same applies to `enable_joy_gyroscope()`.

Reply:
I think we should revert it to the previous naming still, as we have a lot of other methods in Godot that follow the `set_*_enabled()` pattern.